src/main/kotlin/Interfaces.kt

```diff
@@ -7,7 +7,7 @@ interface Payable {
 }
 
 interface Fillable {
-    fun fillTank(station: ShellStation)
+    fun fillTank(station: ServiceStation)
```

Comment:
I know I wrote this sample code with `station: ShellStation`, but we'd rather make it more general. See implementation details below.